README.md

```diff
@@ -1,254 +1,142 @@
-and second for the commit containing that metadata.  The `secure-push` command
-will also create the `rsl` branch if necessary.
+ARGS:
+    <src>     The branch to merge
+    <dest>    The destination branch
 
-```
-git secure-push
-git secure-push HEAD
-git secure-push HEAD:remote_branch
-git secure-push local_branch
-git secure-push local_branch:remote_branch
-```
-
-### secure-fetch
 
-Usage: `git secure-fetch [branch]`
+git-promote --help
 
-Git is vulnerable to metadata attacks as detailed in the [RSL paper](https://www.usenix.org/system/files/conference/usenixsecurity16/sec16_paper_torres-arias.pdf).
-In order to create a signed and replicated state log the `git-secure-push` and
-`git-secure-fetch`/`git-secure-pull` scripts work in concert to create a
-reference state log for branches and tags that can be verified by other
-developers to ensure the repository metadata is consistent (or at least the
-parts known during the last interaction with the Bitbucket or GitHub server are
-unchanged).
 
-The `secure-fetch` subcommand will only fetch the currently checked out branch. It
-will also create a nonce on the `rsl` branch and therefore will prompt for GPG
-signature as part of that commit.
+/// TODO
 
-The `secure-fetch` subcommand will not merge any changes into the local working
-branch. For that you'll either need to use `git merge` explicitly, or use `git
-secure-pull`.
-
-```
-git secure-fetch
 ```
 
-### secure-pull
-
-Usage: `git secure-pull [branch]`
+### Examples
 
-Git is vulnerable to metadata attacks as detailed in the [RSL paper](https://www.usenix.org/system/files/conference/usenixsecurity16/sec16_paper_torres-arias.pdf).
-In order to create a signed and replicated state log the `git-secure-push` and
-`git-secure-fetch`/`git-secure-pull` scripts work in concert to create a
-reference state log for branches and tags that can be verified by other
-developers to ensure the repository metadata is consistent (or at least the
-parts known during the last interaction with the Bitbucket or GitHub server are
-unchanged).
+Suppose you have a product with a devel branch, off which feature branches are
+created, and a master branch, representing the released software.
 
-The `secure-pull` subcommand will only fetch and merge the currently checked out branch. It
-will also create a nonce on the `rsl` branch and therefore will prompt for GPG
-signature as part of that commit.
-
-The `secure-pull` subcommand will only attempt to fast-forward merge any upstream
-changes and will fail if it is unable to do so. If you need a recursive/3-way
-merge, you'll need to explicitly use `git merge`.
-
-```
-git secure-pull
-```
+* Set up an example project
+  ```bash
+  mkdir sample-origin.git
+  cd sample-origin.git
+  git init . --bare
+  cd ..
+  
+  git clone sample-origin.git sample-workspace
+  cd sample-workspace
+  
+  git commit -m "Initial Revision"
+  git secure-push origin master
+  
+  git co -b devel
```

Comment:
Probably best to keep the use of aliases down when we're already introducing new things.